From 43e2760f536737cbd50dd3d8eb7faf39633b9e8c Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston <jeremyhu@apple.com>
Date: Wed, 28 Mar 2012 15:07:51 -0700
Subject: [PATCH 4001/4003] os: Fix regression with FatalError not calling
 va_start

Regression From: a818b305989bbcde4e585112a7ee70cbc0b14a92

Found-by: Colin Harrison <colin.harrison@virgin.net>
Signed-off-by: Jeremy Huddleston <jeremyhu@apple.com>
(cherry picked from commit 817cec8ffd6cdb1abbcc00decd43569d44801764)
---
 os/log.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/os/log.c b/os/log.c
index d913449..501fd45 100644
--- a/os/log.c
+++ b/os/log.c
@@ -596,6 +596,8 @@ FatalError(const char *f, ...)
     else
         ErrorF("\nFatal server error:\n");
 
+    va_start(args, f);
+
     /* Make a copy for OsVendorFatalError */
     va_copy(args2, args);
 
-- 
1.7.10

